cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -672,7 +675,12 @@ def make_resource(self, data, **kwargs):
 class S3AccessSchema(BaseSchema):
     """Represent the schema of S3 access."""
 
-    bucket_name = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.SUPPORTED})
+    bucket_name = fields.Str(
+        required=True,
+        metadata={"update_policy": UpdatePolicy.SUPPORTED},
+        validate=validate.Regexp(r"[^\\/]*$", error="Bucket name cannot contain any back/forward slash."),
```

Comment:
we might use a positive regex: [a-z0-9-\.]*

See https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html

Reply:
Done